### PR TITLE
Remove unwanted text from UX.

### PR DESCRIPTION
**Pull request by usingtechnology**
Add Tenant name to Sidebar Title.
Remove unused functions.
Tighten up login to use not only token but a fetched tenant before loading (needed for tenant name).

JIRA TICKETS

1. https://apps.nrs.gov.bc.ca/int/jira/browse/TRACTION-392 (remove copyright, owner text from footer)
2. https://apps.nrs.gov.bc.ca/int/jira/browse/TRACTION-393 (remove home disclaimer etc from footer replace with powered by traction)
3. https://apps.nrs.gov.bc.ca/int/jira/browse/TRACTION-395 (tenant name instead of traction on sidebar top)
4. https://apps.nrs.gov.bc.ca/int/jira/browse/TRACTION-390 (pin footer)
5. https://apps.nrs.gov.bc.ca/int/jira/browse/TRACTION-394 (kind of removed text, added a logo but dashboard details not confirmed yet)

Signed-off-by: Jason Sherman <tools@usingtechnolo.gy>

### CODE REVIEW

File: services/tenant-ui/frontend/src/App.vue [hunk 1]
```diff
@@ -1,21 +1,20 @@
 <script setup lang="ts">
 import AppLayout from "./components/layout/AppLayout.vue";
 import Login from "./components/Login.vue";
-import {onMounted} from "vue";
+import {computed, onMounted} from "vue";
```
Comment (loneil) [inline services/tenant-ui/frontend/src/App.vue:4]: Computed looks unused?

**Review by loneil — COMMENTED** [review]
Think there's a size/overflow issue, can scroll off to the right (Chrome at least)
image

**Comment by usingtechnology** [comment]
Footer clean up worked... big thanks 🙌

File: services/tenant-ui/frontend/src/App.vue (hunk 1 above)
Comment (usingtechnology) [reply services/tenant-ui/frontend/src/App.vue]: removed

**Review by loneil — APPROVED** [review]
Great stuff

**Pull request merged**